packages/server/src/schema.js

```diff
@@ -107,6 +107,17 @@ const schema = new graphql.GraphQLSchema({
         resolve: resolvers.agentMeasurables.all,
       },
 
+      agentChannels: {
+        type: types.agentChannels.agentChannelsConnection,
+        args: {
+          ...types.common.connectionArguments,
+          minNumberOfPredictions: { type: graphql.GraphQLInt },
```

Comment:
So the use case here is to filter by the minNumberOfPredictions? Why is this needed? I could imagine it being useful later on, not sure about now.

Reply:
There is no huge need to write logic with redundant conditions, instead of this we are able to give to the clients (UI) possibilities to solve what and how they want to get. It is not pattern, but I believes it is more flexible then hard code.

Reply:
ok